WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Multi-package/SinglePackageHazmatDeclaration.swift

```diff
@@ -78,47 +79,69 @@ struct SinglePackageHazmatDeclaration: View {
                 .calloutStyle()
 
             Spacer()
-            Text(Localization.hazmatInstructionsSecondSection)
-                .calloutStyle()
+            createText(withLink: Localization.hazmatInstructionsSecondSectionLink,
+                       url: WooConstants.URLs.uspsInstructions.asURL(),
+                       content: Localization.hazmatInstructionsSecondSection)
 
             Spacer()
-            Text(Localization.hazmatInstructionsThirdSection)
-                .calloutStyle()
+            createText(withLink: Localization.hazmatInstructionsThirdSectionLink,
+                       url: WooConstants.URLs.uspsSearchTool.asURL(),
+                       content: Localization.hazmatInstructionsThirdSection)
 
             Spacer()
-            Text(Localization.hazmatInstructionsFourthSection)
-                .calloutStyle()
+            createText(withLink: Localization.hazmatInstructionsFourthSectionLink,
+                       url: WooConstants.URLs.dhlExpressInstructions.asURL(),
+                       content: Localization.hazmatInstructionsFourthSection)
 
             Spacer()
         }
         .padding(.leading, Constants.horizontalPadding)
         .padding(.trailing, Constants.longTextTrailingPadding)
     }
+
+    func createText(withLink linkText: String, url: URL, content: String) -> some View {
+        Text(.init("\(content) [\(linkText).](\(url))"))
```

Comment:
This might not work for all languages, unless we make sure the link text is always at the end. Alternatively, we can use `LearnMoreAttributedText` to show a text with a tappable link with the preview as an example

Reply:
True! I tried to explore some ways to use string interpolation to achieve this to avoid internationalization issues but no luck, but your suggestion over the `LearnMoreAttributedText` was exactly the knowledge I needed to fix this, thank you so much! It's now fixed here in https://github.com/woocommerce/woocommerce-ios/pull/10598/commits/d57db22b94cf37db562c528101e813e6f42ae3f6